### DISCUSSION TITLE
Headphones with usb-c jack

### DISCUSSION THREAD

**Discussion by thomasbonset** [OnionUI/Onion#1694] [Ideas]
Hello folks,
what do you think about making listening with wired headphones functional through the usb-c port. I don't know how feasible it is, but it could be a nice feature. 
thanks

**Comment by alessdangelo** [comment]
If I remember correctly, the usb-c port on the Miyoo minis does not read data and is only used as a charging port

**Reply by thomasbonset** [reply]
oh that's sad😒
thanks for answering by the way.
Have a nice day
